gulpfile.js

```diff
@@ -121,9 +122,28 @@ gulp.task('copy-test-dirs', function() {
 });
 
 gulp.task('copy-images', function () {
+  var TO_JPEG = {
+    format: 'jpg',
+    formatParams: {
+      quality: 70
+    }
+  };
   return gulp.src(IMG_FILE_TYPES, {
     base: '.'
-  }).pipe(gulp.dest('./dist'));
+  }).pipe(imageConverter({
+    files: {
+      'img/pages/home/hero-unit@2x.png': TO_JPEG,
+      'img/pages/home/hero-unit.png': TO_JPEG,
+      'img/pages/activities/hero-teaching-activities@2x.png': TO_JPEG,
+      'img/pages/activities/hero-teaching-activities.png': TO_JPEG,
+      'img/pages/clubs/hero-clubs@2x.png': TO_JPEG,
+      'img/pages/clubs/hero-clubs.png': TO_JPEG,
+      'img/pages/events/hero-events@2x.png': TO_JPEG,
+      'img/pages/events/hero-events.png': TO_JPEG,
+      'img/pages/teach-like-mozilla/hero-teach-like-mozilla@2x.png': TO_JPEG,
+      'img/pages/teach-like-mozilla/hero-teach-like-mozilla.png': TO_JPEG
+    }
```

Comment:
OOOoooooooooo good idea! 

Hmm do you mind if we file it as a separate issue though, and just land this as-is? I'm worried adding support for filename globbing could take a while and this PR is blocking some other optimizations I'd like to add, like #1030.